simplex/messages.go

```diff
@@ -224,6 +224,10 @@ func quorumRoundToP2P(qr *simplex.VerifiedQuorumRound) (*p2p.QuorumRound, error)
 		}
 	}
 	if qr.Finalization != nil {
+		// This can only happen if the finalization of the genesis block is being sent
```

Comment:
So, this code path involves us sending a message to a remote node.

Given that, why return an error? why just not return nil QC bytes?